### ISSUE TITLE
[FEATURE] Views for Uno WinUI 3

### ISSUE THREAD

**Issue by beto-rodriguez** [mono/SkiaSharp#1787]
**Is your feature request related to a problem? Please describe.**

I re-opened this issue from #1742 to track this implementation @jeromelaban.

Please add Views for Uno WinUI 3 target. As of now I can't use SkiaSharp.Views.Uno in project created with unoapp-winui, nor can I install the SkiaSharp.Views.WinUI into the platform implementations.

**Describe the solution you'd like**

Add SkiaSharp.Views.UnoWinUI nuget which will depend on Uno.WinUI rather then Uno.UI

**Comment by anpin** [comment]
hi @jeromelaban, is there any update on this?

**Comment by jeromelaban** [comment]
Not at this point, but pieces are in place for the work to continue.

**Comment by anpin** [comment]
I've started putting this together

**Comment by anpin** [comment]
Unfortunately experience with bleeding edge tooling was frustrating for me. At first both ```cake``` and ```vs2022-preview``` were telling me that some workloads are not installed (e.g. macos or ios) even though I could see them in ```dotnet``` and they were updated, so I had to reinstall ```dotnet``` workloads multiple times and removed older sdk's until msbuild somehow recognized all workloads and started to build. Then I've put up new view projects and samples targeting Uno.WinUI and referencing all the same files as current Uno projects, but with conditional compilation to allow different ```using``` references for WinUI. Then I realized that I shouldn't just copy project structure for existing Uno views and instead should create a multi-targeted project like Maui views has, so I redo that part. At this point I was faced with ```Unable to map the following RuntimeIdentifier values to a target architecture: win10-arm-aot;win10-arm64-aot;win10-x86-aot;win10-x64-aot``` which I couldn't really understand how to fix and VS2022 preview update has arrived, so I decided to try it out. So now ```msbuild``` doesn't recognize installed workloads again. I tried to remove not preview version of VS2022 as well as ```dotnet 6.0.200 sdk``` installed from binary, so only the VS 17.2.0 Preview 1.0 with it's bundled sdk would be left on my machine, but it still shows that workloads are missing even though "dotnet workload list" shows every needed piece. As I'm writing this I see that there is now dotnet 7 preview, so I'll try to use that instead of 6.0.200. 
(P.S. plus the same evening I wrote my previous comment I've got covid with high fewer on next day and I'm not yet fully recovered, so my judgement is a little off)  
![image](https://user-images.githubusercontent.com/6060545/154671175-84e64ce9-1b95-4ded-88a7-a89d62ead763.png)
